src/inmanta/app.py

```
@@ -620,6 +621,74 @@ def export(options: argparse.Namespace) -> None:
     summary_reporter.print_summary_and_exit(show_stack_traces=options.errors)
 
 
+def validate_logging_config_parser_config(
+    parser: argparse.ArgumentParser, parent_parsers: abc.Sequence[ArgumentParser]
+) -> None:
+    """
+    Config parser for the validate-logging-config command.
+    """
+    parser.add_argument(
+        "filename",
+        help="Path to the logging configuration file.",
+    )
+    parser.add_argument(
+        "-e",
+        dest="environment",
+        help="The environment id to be used as context variable in logging config templates.",
+    )
```

I wonder if we should default to random uuid?